code/unclassified/src/average/average.nim

```diff
@@ -0,0 +1,7 @@
+func average(arr: seq[int]): int =
+    for i in arr:
+      result += i
```

Comment:
This function doesn't compute the average. It only computes the sum of the sequence. Please fix this.

Reply:
Sorry my bad. I made the changes and also added a `isMainModule`